### ISSUE TITLE
Deploy relayer fee upgrade to testnet

### ISSUE THREAD

**Issue by LayneHaber** [connext/monorepo#3811]
## Problem
_What problem are we solving and why does it matter, ser?_ 

Need to deploy the relayer fee upgrade to testnet.

## Proposed Solution
_Thoughts on solution design. We should do <insert idea> to solve this problem._

Blocked by having a try-catch to support "backwards compatible" queries

- [x] Load test staging
- [x] Reviews of [upgrade doc](https://www.notion.so/connext/Relayer-Fee-Upgrade-4fd9b7038b64488881629343f7ce7fcf)
- [x] Propose upgrade (dont accept) -- @LayneHaber 
- [x] Separate live environments #3834 -- @LayneHaber 
- [x] Merge from `staging` into `testnet-prod` + router release with #3831 -- @preethamr 
- [x] Deploy sequencer, lighthouse, watcher, etc. (all offchain connext-run agents) to testnet prod
- [x] Deploy router(s)
- [x] Check logs to ensure everything spins up properly in testnet
- [x] Publish sdk @ `2.0.0-alpha.0` -- @just-a-node 
- [x] Start load test on testnet -- @preethamr 
- [x] Upgrade contract -- @LayneHaber 
- [x] Deploy subgraphs (agents will break here) -- @LayneHaber 
~~- [ ] Deploy UI -- @LayneHaber~~
~~- [ ] Deploy Connextscan -- @LayneHaber~~
- [x] Propose mainnet upgrade IFF load tests successful && new feature works
- [x] Announce router version upgrade

## Acceptance Criteria
_From an end tester and/or user perspective, the below criteria must be met to consider this done:_

- [x] Can bridge with relayer fee on testnet bridge ui
- [x] 100% transfer success rate on testnet load UI
- [x] Mainnet upgrade proposed

## Notes on Contract Deployment

To deploy the upgrade, do the following:

1. Ensure the `BridgeFacet.json` contract deployment record is deleted if you want a fresh deploy
2. Verify your `.env` has the following:
```env
NETWORK= # "mainnet" or "testnet"
ENV= # "staging" or "production"
MNEMONIC= # deployer
```

3. Run the upgrade tests (upgrading will fail if you haven't):
```sh
# Run the following to simulate upgrade against forks
ywc test:upgrade
```
**Note:** you may have to update the `FORK_BLOCKS` to be after most recent upgrade performed on testnet contracts

4. Deploy + propose the upgrade:
```sh
# Run the following to deploy the relayer fee upgrade + propose it to your contract
ywc hardhat deploy --tags "relayer-fee-upgrade" --network "$NETWORK_NAME"
```

**Comment by LayneHaber** [comment]
contract transactions:

### Proposals
- goerli: 0x0cbd12df40cf9d800c74f435eb29092c66ae5d5c7ed5470b4ab85ce829d28d8c
- opti-goerli: 0xed7be4469a7a4e632695b45d0e5c77325aa729f48aac56626797a00c80643930
- mumbai: 0xc6f15f7ddfc96bb24165f80fff236ccd236941e3fd26e3b56eddc248e49a7754
- consensys: 0x85de571687e2b0c13bc13fb6a22996592ff87b7889e45282b445233ded3d1ae4
- arb-goerli: 0x763aec3ae8a0ca38c1e8e1cb0e521b61808cc43d60fa91ada59aea6f0508f063

### Upgrades
- goerli: 0xbf892f9e806e8c3f77581b56a99db2e3cf4e02c7de157b1a7b3266b07017fc31
- opti-goerli: 0x607b55a0e759fae5837565d9c4a05ce498360015727a6d269fa40a392931c55a
- mumbai: 0x523b29ee302c0fdd74585784fba824613c5a0954feacd0f491548728e2f78e8c
- consensys: 0x5f7da7257281ad325488a83382fc471edd6bb0337e5c1bc96cb2dfc0769b56da
- arb-goerli: 0x7b0245b115d5a32441ddb6de8a5123e96bc9bd89693d82ece00f181be947bfa9

**Comment by LayneHaber** [comment]
sdk released @ `2.0.0-alpha.0`: https://github.com/connext/monorepo/releases
